test_sets/ExternalUserConsentStatus_Informing_HMI.txt

```diff
@@ -1,9 +1,17 @@
-./test_scripts/Polices/External_UCS_Informing_HMI/005_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_lower_from_OnAppPermissionConsent_notification_Entity_Status_OFF
-./test_scripts/Polices/External_UCS_Informing_HMI/006_ATF_P_GetListofPermissions_with_appID_all_ecs_upper_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_upper_from_OnAppPermissionConsent_notification_Entity_Status_OFF
-./test_scripts/Polices/External_UCS_Informing_HMI/007_ATF_P_GetListofPermissions_with_appID_not_stored_ecs.lua Check_that_SDL_returns_empty_array_appID_no_stored_ecs
-./test_scripts/Polices/External_UCS_Informing_HMI/008_ATF_P_GetListofPermissions_no_appID_stored_ecs_present.lua Check_that_SDL_responds_with_values_from_ecs_no_appID
-./test_scripts/Polices/External_UCS_Informing_HMI/009_ATF_P_GetListofPermissions_no_appID_not_stored_ecs.lua Check_that_SDL_returns_empty_array_no_appID_no_stored_ecs
\ No newline at end of file
+./test_scripts/Polices/External_UCS_Informing_HMI/004_ATF_P_GetListofPermissions_with_appID_fake_param_cut_EntitySatus_ON.lua Check_that_SDL_returns_all_params_from_OnAppPermissionConsent_notification_and_cuts_fake_param_Entity_Status_ON
+./test_scripts/Polices/External_UCS_Informing_HMI/005_ATF_P_GetListofPermissions_with_appID_all_ecs_in_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_in_bound_from_OnAppPermissionConsent_notification_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/006_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_lower_from_OnAppPermissionConsent_notification_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/007_ATF_P_GetListofPermissions_with_appID_all_ecs_upper_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_upper_from_OnAppPermissionConsent_notification_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/008_ATF_P_GetListofPermissions_with_appID_fake_param_cut_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_from_OnAppPermissionConsent_notification_and_cuts_fake_param_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/009_ATF_P_GetListofPermissions_with_appID_not_stored_ecs.lua Check_that_SDL_returns_empty_array_appID_no_stored_ecs
+./test_scripts/Polices/External_UCS_Informing_HMI/010_ATF_P_GetListofPermissions_no_appID_stored_ecs_present.lua Check_that_SDL_responds_with_values_from_ecs_no_appID
+./test_scripts/Polices/External_UCS_Informing_HMI/011_ATF_P_GetListofPermissions_no_appID_not_stored_ecs.lua Check_that_SDL_returns_empty_array_no_appID_no_stored_ecs
+./test_scripts/Polices/External_UCS_Informing_HMI/012_ATF_N_GetListofPermissions_no_appID_no_app_registered.lua Check_that_SDL_returns_empty_array_when_no_app_registered

```

Comment:
@maria-mih, SDL will return empty array only if 'externalConsentStatus' is not stored.
At the moment you should have sequence:
1. externalConsentStatus - not stored
2. HMI->SDL: GetListOfPermissions (appID), appID is for application that is not registered
=> SDL will accept GetListOfPermissions (without appID)
Expected: SDL -> HMI: GetListOfPermissions ("externalConsentStatus": [ ]) 

You should cover also case, when 'externalConsentStatus' is stored.